src/EditorFeatures/CSharpTest/ExtractClass/ExtractClassTests.cs

```diff
@@ -0,0 +1,1596 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeRefactorings;
+using Microsoft.CodeAnalysis.CSharp.CodeRefactorings.ExtractClass;
+using Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.CodeRefactorings;
+using Microsoft.CodeAnalysis.ExtractClass;
+using Microsoft.CodeAnalysis.PullMemberUp;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.ExtractClass
+{
+    public class ExtractClassTests : AbstractCSharpCodeActionTest
+    {
+        private Task TestAsync(
+            string input,
+            string expected,
+            IEnumerable<(string name, bool makeAbstract)> dialogSelection = null,
+            bool sameFile = false,
+            bool expectNullMember = false,
```

Comment:
what is a null member?

Reply:
No selected member, for when a class declaration is the input node. Changed to `isClassDeclarationSelection`